src/jit/simd.cpp

```diff
@@ -77,7 +77,8 @@ int Compiler::getSIMDVectorLength(CORINFO_CLASS_HANDLE typeHnd)
 //
 int Compiler::getSIMDTypeAlignment(var_types simdType)
```

Comment:
>getSIMDTypeAlignment [](start = 14, length = 20)

FYI: Please note that this routine gets used in lclvars.cpp::lvaAllocLocalAndSetVirtualOffset() and that logic is enabled only for 64-bit.  There is another #define that governs whether SIMD locals are aligned ALIGN_SIMD_TYPES (in target.h), which is currently defined only for amd64.

isSIMDTypeLocalAligned() defined in compiler.h is the one used by codegen to check whether a SIMD local is aligned.